lib/web/question_router.ts

```diff
@@ -667,162 +725,196 @@ async function getQuestionsUserCreatedOrForecastedOnOrIsSharedWith(
-          resolution: null,
-          resolveBy: {
-            lte: new Date(),
-          }
-        } : {},
-        input.extraFilters?.resolvingSoon ? {
-          resolveBy: {
-            gte: new Date(),
-          },
-          resolution: null,
-        } : {},
-        input.extraFilters?.filterTagIds ? {
-          tags: {
-            some: {
-              id: {
-                in: input.extraFilters.filterTagIds
-              }
+        input.extraFilters?.resolved
+          ? {
+              resolution: {
+                not: null,
+              },
+            }
+          : {},
+        input.extraFilters?.readyToResolve
+          ? {
+              resolution: null,
+              resolveBy: {
+                lte: new Date(),
+              },
+            }
+          : {},
+        input.extraFilters?.resolvingSoon
+          ? {
+              resolveBy: {
+                gte: new Date(),
+              },
+              resolution: null,
+            }
+          : {},
+        input.extraFilters?.filterTagIds
+          ? {
+              tags: {
+                some: {
+                  id: {
+                    in: input.extraFilters.filterTagIds,
+                  },
+                },
+              },
             }
-          }
-        } : {},
-      ]
+          : {},
+      ],
     },
     include: questionIncludes(ctx.userId),
   })
 
   return {
     items: questions
-      .map(q => scrubHiddenForecastsFromQuestion(q, ctx.userId))
+      .map((q) => scrubHiddenForecastsFromQuestion(q, ctx.userId))
       // don't include the extra one - it's just to see if there's another page
       .slice(0, limit),
 
-    nextCursor: (questions.length > limit) ? skip + limit : undefined,
+    nextCursor: questions.length > limit ? skip + limit : undefined,
   }
 }
 
-export async function emailNewlySharedWithUsers(newlySharedWith: string[], question: QuestionWithUserAndSharedWith) {
-  await Promise.all(newlySharedWith.map(async (email) => {
-    const author = question.user.name || question.user.email
-    await sendEmail({
-      to: email,
-      subject: `${author} shared a prediction with you`,
-      textBody: `"${question.title}"`,
-      htmlBody: `<p>${author} shared a prediction with you: <b>${getHtmlLinkQuestionTitle(question)}</b></p>
-<p><a href=${getQuestionUrl(question)}>See ${author}'s prediction and add your own on Fatebook.</a></p>
-${fatebookEmailFooter(email)}`
+export async function emailNewlySharedWithUsers(
+  newlySharedWith: string[],
+  question: QuestionWithUserAndSharedWith
+) {
+  await Promise.all(
+    newlySharedWith.map(async (email) => {
+      const author = question.user.name || question.user.email
+      await sendEmail({
+        to: email,
+        subject: `${author} shared a prediction with you`,
+        textBody: `"${question.title}"`,
+        htmlBody: `<p>${author} shared a prediction with you: <b>${getHtmlLinkQuestionTitle(
+          question
```

Comment:
I think I prefer the previous format here but not a big deal at all, happy to change to this

Reply:
I think I figured out what's going on with these formatting changes, your prettier formatter may be pointing to a system wide config that's different to the .prettierrc.json in the workspace root? I also prefer a wider print width of 120, maybe we want to update it?